notifier/events/voyager_is_processing_whole_request_event.py

```diff
@@ -22,7 +22,5 @@ def __str__(self):
         TODO: Fill the content of the email
         :return: email body
         """
-        body = "Project {igo_request_id} is successfully running. All samples matched.".format(
-            igo_request_id=self.request_id
-        )
+        body = f"We have received the request to run {self.request_id}. Please review this information carefully if this is not what you expected contact us. Results can be expected in 10-14 days."
```

Comment:
Can you change this to

"We have received the request to run {self.request_id} and will begin processing them soon. Results typically take 10-14 days to generate.

Please review this information carefully; if anything in this project is not what you expected, contact us and we'll be happy to help."

Reply:
Changed